src/frontend/komponenter/Task/TaskMedCallId.tsx

```diff
@@ -0,0 +1,51 @@
+import { Alert, Heading } from '@navikt/ds-react';
+import { RessursStatus } from '@navikt/familie-typer';
+import * as React from 'react';
+import { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { useServiceContext } from '../ServiceContext';
+import { useTaskContext } from '../TaskProvider';
+import TaskListe from './TaskListe';
+import TaskPanel from './TaskPanel';
+
+const TaskMedId: React.FC = () => {
```

Comment:
```suggestion
const TasksMedCallId: React.FC = () => {
```

Burde denne komponenten hete det samme som filen? Og burde det være tasks istedet for task?

Reply:
Jepp